### PR TITLE
Add doc for allowedFlexVolumes

### PR DESCRIPTION
**Pull request by wanghaoran1988**
Closes: #6447

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/6563)

### CODE REVIEW

File: docs/concepts/policy/pod-security-policy.md [hunk 1]
```diff
@@ -162,6 +163,28 @@ spec:
     - pathPrefix: "/foo"
 ```
 
+### AllowedFlexVolumes
+
+This specifies a whitelist of flex volume drivers that are allowed to be used
+by flexVolume. An empty list means there is no restriction on the drivers. we
+need make sure `volumes` contains the `flexVolume` volume type, if it doesn't
```
Comment (tengqm) [inline docs/concepts/policy/pod-security-policy.md]: > we need make sure

Please make sure .... No flex volume driver is allowed otherwise.

**Comment by php-coder** [comment]
CC @liggitt @tallclair

File: docs/concepts/policy/pod-security-policy.md [hunk 2]
```diff
@@ -162,6 +163,28 @@ spec:
...
+, no flex volume driver is allowed.
```
Comment (php-coder) [inline docs/concepts/policy/pod-security-policy.md]: Could you put a comma right after the word (instead of on a new line)?

**Comment by wanghaoran1988** [comment]
@php-coder @tengqm Update, PTAL

**Comment by wanghaoran1988** [comment]
rebased

**Pull request merged**